src/widgets/visit/visit-dashboard.component.test.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import { BrowserRouter } from "react-router-dom";
+import VisitDashboard from "./visit-dashboard.component";
+import {
+  render,
+  screen,
+  fireEvent,
+  RenderResult,
+  findByRole,
+  findByText
+} from "@testing-library/react";
+import {
+  getCurrentPatientUuid,
+  openmrsObservableFetch,
+  useCurrentPatient,
+  openmrsFetch
+} from "@openmrs/esm-api";
+import { of } from "rxjs";
+import {
+  mockVisitTypesDataResponse,
+  mockVisits
+} from "../../../__mocks__/visits.mock";
+import { mockLocationsDataResponse } from "../../../__mocks__/location.mock";
+import { mockSessionDataResponse } from "../../../__mocks__/session.mock";
+import { mockPatient } from "../../../__mocks__/patient.mock";
+import { async } from "rxjs/internal/scheduler/async";
+// import { getPatientVisits } from "./visit-resource";
+
+const mockGetCurrentPatientUuid = getCurrentPatientUuid as jest.Mock;
+const mockOpenmrsObservableFetch = openmrsObservableFetch as jest.Mock;
+const mockUseCurrentPatient = useCurrentPatient as jest.Mock;
+const mockOpenmrsFetch = openmrsFetch as jest.Mock;
+
+jest.mock("@openmrs/esm-api", () => ({
+  getCurrentPatientUuid: jest.fn(),
+  openmrsObservableFetch: jest.fn(),
+  useCurrentPatient: jest.fn(),
+  openmrsFetch: jest.fn(),
+  fhirBaseUrl: "/ws/fhir2"
+}));
+
+// jest.mock("./visit-resource", () => ({
+//   getPatientVisits: jest.fn()
+// }));
+
+describe("VisitDashboard", () => {
+  let patientUuid = "some-patient-uuid";
+  let patient = mockPatient;
+  let wrapper: RenderResult;
```

Comment:
Instead of using the result of `render`, let's use `screen`. Following https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen